src/components/AppLayout/Header/components/ProviderInfo/ProviderAccessible.tsx

```diff
@@ -90,16 +85,22 @@ const ProviderInfo = ({ connected, provider, userAddress }: ProviderInfoProps):
           {provider}
           {chain?.chainName && ` @ ${chain.chainName}`}
         </Paragraph>
-        <div className={classes.providerContainer}>
+        <div>
           {connected ? (
-            <PrefixedEthHashInfo
-              hash={userAddress}
-              shortenHash={4}
-              showAvatar
-              avatarSize="xs"
-              textColor={addressColor}
-              textSize="sm"
-            />
+            ensName ? (
+              <Text strong size="sm">
```

Comment:
11px font size and bold is a bit hard to read. We could tweak it in the future

Reply:
Agree. I'll keep it as "sm" at this time, the same the hash font.